route/build/src/var_lookup.f90

```diff
@@ -13,256 +13,266 @@ MODULE var_lookup
-  integer(i4b)     :: H06_frac_Sdead   = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_E_rel_ini    = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Jan        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Feb        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Mar        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Apr        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_May        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Jun        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Jul        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Aug        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Sep        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Oct        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Nov        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_Dec        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Jan        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Feb        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Mar        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Apr        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_May        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Jun        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Jul        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Aug        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Sep        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Oct        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Nov        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_Dec        = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_purpose      = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_mem_F      = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_mem_F      = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_I_mem_L      = integerMissing  ! Hanasaki 2006 parameter
-  integer(i4b)     :: H06_D_mem_L      = integerMissing  ! Hanasaki 2006 parameter
-
+  integer(i4b)     :: basUnderLake     = integerMissing  ! 11. Area of basin under lake (m2)
+  integer(i4b)     :: rchUnderLake     = integerMissing  ! 12. Length of reach under lake (m)
+  ! Doll 2003 parameter (Natural lake outflow)
+  integer(i4b)     :: D03_MaxStorage   = integerMissing  ! 13. Lake maximum volume (m3)
+  integer(i4b)     :: D03_Coefficient  = integerMissing  ! 14.
```

Comment:
At some point the description of these variables needs a better description of what they each are.

Reply:
@ekluzek I fully agree. I will come back to this in October and I'll populate the explanation in var_lookup.f90
Just to say that the Hanasaki parameters in mizuRoute might be a different from original formulation as we have added memory as well. Also we have introduce some parameters, such as power, etc, which was hard coded in original Hanasaki formulation and that explains the large number of parameter in comparison to original Hanasaki.

Reply:
Sounds great @ShervanGharari. Thanks for adding that update. I look forward to your working on this.